bin/bundle_spec

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+docker run --rm \
+    -v ${PWD}:/code \
+    node:latest \
+    /bin/bash -c "npm i -g @stoplight/spectral && npm i -g swagger-cli && swagger-cli bundle /code/spec/conjur-openapi.yml --outfile /code/spec.yml --type yaml"
```

Comment:
(nit) In one of our next PRs, let's make this separate lines so it's easier to modify
However, this isn't a blocker.

I recommend something like this:
```
docker run --rm \
    -v "$(pwd)":/code \
    node:latest \
    /bin/bash -c """
    npm i -g @stoplight/spectral
    npm i -g swagger-cli
    swagger-cli bundle '/code/spec/conjur-openapi.yml' --outfile '/code/spec.yml' --type yaml
    """
```